✨ Add the option to ignore query params in caching (#4)
store2be/nginx-reverse-proxy: README.md

# Example
CACHE=true
```
### CACHE_IGNORE_QUERY_PARAMS

> You can configure caching to ignore query parameters and only cache according to the URL.

Reviewer: ```suggestion
You can configure caching to ignore query parameters and only cache according to the path of the URL.
```

docker run --privileged -p 80:80 --rm -e PROXY_URL=example.com nginx-reverse-proxy strace nginx-debug -g 'daemon off;'
docker exec -ti $(docker ps | grep nginx-reverse-proxy | awk '{print $1}') tcpdump not port 22 -vvv -s0 -q -XXX
```

## Releasing

### Before merging to master

> Make sure you update CHANGELOG.md and README.md when appropriate.

Reviewer: ```suggestion
Make sure you update the [CHANGELOG](CHANGELOG.md) and the [README](README.md) when appropriate.
```

*CHANGELOG.md entry format:*

```
## [0.0.0] - 1970-01-01
- ✨ Add a new feature
```

Bump version according to [SemVer](https://semver.org/).

### After merging to master

> Tag the head with the new version, this should generate an image with the version as a tag.

Reviewer: ```suggestion
Tag the head with the new version and push it to Github. This then automatically generates the new image on quay.io with the version as a tag.
```
